server/coordinator/procedure/storage.go

```diff
@@ -0,0 +1,21 @@
+package procedure
+
+import (
+	"context"
+)
+
+type Write interface {
+	CreateOrUpdate(ctx context.Context, meta *Meta) error
```

Comment:
Because the `state` parameter is used in  the `list` func, it should also be used as an independent parameter here, or the `state` parameter in the `list` func should be removed directly